dev-docs/bidders/exadsadserver.md

```diff
@@ -0,0 +1,484 @@
+---
+layout: bidder
+title: EXADS
+description: EXADS Bidder Adapter
+pbjs: true
+biddercode: exadsadserver
+gvl_id: 1084
+tcfeu_supported: true
+media_types: banner, video, native
```

Comment:
Please add

- `multiformat_supported` : will-bid-on-any, will-bid-on-one, will-not-bid
- `dsa_supported: true`

as you support multiple formats and apparently DSA

Reply:
done